### ISSUE TITLE
HTTP Error code 410

### ISSUE THREAD

**Issue by jxmesth** [instaloader/instaloader#703]
**Describe the bug**
When trying to download highlights from a profile, Instagram gives an HTTP error. I'm pretty sure this has got nothing to do with being rate limited or blocked by Instagram as I have tried to download the profile using different accounts and IP addresses.

**To Reproduce**
The command I have used is `instaloader PROFILE --login=USER --stories --tagged --highlights --igtv`

**Expected behavior**
Downloads the entire profile.

**Error messages and tracebacks**
Instaloader gives the following errors after downloading a few posts and highlights:

```
Download highlights "ON TOUR" from user PROFILE: download_pic(): HTTP error code 410.
Download highlights "ON TOUR" from user PROFILE: download_pic(): HTTP error code 410.
Download highlights "ON TOUR" from user PROFILE: download_pic(): HTTP error code 410.
Download highlights "ON TOUR" from user PROFILE: download_pic(): HTTP error code 410.
Download highlights "ON TOUR" from user PROFILE: download_pic(): HTTP error code 410.
Download highlights "ON TOUR" from user PROFILE: download_pic(): HTTP error code 410.
```

**Instaloader version**
Version 4.4.4

**Comment by Thammus** [comment]
The error code 410 that is sent by Instagram means that the resource is gone and Instaloader skips the respective posts. Are you able to view this highlight in the official Instagram app or the web interface?

**Comment by jxmesth** [comment]
Yupp, I'm able to see the highlight on Instagram but Instaloader gives an error when trying to download the highlight.

**Comment by jxmesth** [comment]
Is there a way to force Instaloader to ignore the error and continue with the rest of the downloads?

**Comment by ahogan-intertel** [comment]
Having the same issue, causes instaloader to fail when it reaches a 410. 

`download_pic(): HTTP error code 410. [retrying; skip with ^C]`

**Comment by Thammus** [comment]
> Yupp, I'm able to see the highlight on Instagram but Instaloader gives an error when trying to download the highlight.

Are there any highlights visible in the web interface that are not downloaded by Instaloader?

> Is there a way to force Instaloader to ignore the error and continue with the rest of the downloads?

After getting an error for the same download attempt several times, Instaloader skips it and continues with the next one. The same logic gets triggered when pressing CTRL+C.

**Comment by jxmesth** [comment]
> Are there any highlights visible in the web interface that are not downloaded by Instaloader?

There are a few highlights that Instaloader misses. This might be because Instaloader stops downloading after the error comes up.

**Comment by Thammus** [comment]
Could you please provide a profile (preferably a public one) on which this error occurs to reproduce it?

**Comment by jxmesth** [comment]
Weirdly, I haven't seen this error come up for any public profiles that I've downloaded. In fact, out of all the profiles I've downloaded I've only gotten this specific error for one profile, a private one, so I'm not sure if I have a profile on which this issue can be replicated at the moment. 

I'll be closing this issue for now since it can't be replicated and I'll reopen it if I come across another profile that gives the same error. Thanks for your help and keep up the amazing work!! :)
